### ISSUE TITLE
Improve che-devfile-registry CI

### ISSUE THREAD

**Issue by l0rd** [eclipse-che/che#13813]
### Description

The problems with che-devfile-registry CI are that the community images (centos) are pushed to `quay.io/openshiftio` instead of `quay.io/eclipse-che`, no image is tagged as nightly and there are no periodic (nightly) builds.

To address that we should: 

- [x] Update che-devfile-registry-master CentOS CI job to push rhel images to `quay.io/openshiftio` and centos images to `quay.io/eclipse-che` PR - eclipse/che-devfile-registry#31
- [x] Update che-devfile-registry-periodic CentOS CI job to build the devfile registry image (centos only) and tag it as nightly

This is related to the "Update release process to include new repositories" enhancement of [Che 7 endgame](https://github.com/eclipse/che/issues/13637)

**Comment by ibuziuk** [comment]
PR for first check box - https://github.com/eclipse/che-devfile-registry/pull/31

**Comment by ibuziuk** [comment]
@l0rd and looks like we have first successful build to https://quay.io/repository/eclipse-che/che-devfile-registry?tab=tags \o/

**Comment by ibuziuk** [comment]
PRs for the second checkbox have been sent and merged `Update che-devfile-registry-periodic CentOS CI job to build the devfile registry image (centos only) and tag it as nightly` : 
- https://github.com/eclipse/che-devfile-registry/pull/35
- https://github.com/openshiftio/openshiftio-cico-jobs/pull/1037 

The nightly job has been created https://ci.centos.org/job/devtools-che-devfile-registry-nightly/ but it currently fails with:

> FATAL: java.nio.channels.ClosedChannelException
> java.nio.channels.ClosedChannelException
> Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to JNLP4-connect connection from slave04.ci.centos.org/172.19.4.20:41706

This seems to be related to `ci.centos` outage and all the slaves are currently offline.

@rhopp @Katka92 hope you will be able to finalize the job  '{ci_project}-{git_repo}-nightly' template if needed

**Comment by ibuziuk** [comment]
we have the first `nighly` image now \o/ https://quay.io/repository/eclipse/che-devfile-registry?tab=tags

I have sent PR for making the job really "nightly" since previously it was executed every 30 mins for testing purposes - https://github.com/openshiftio/openshiftio-cico-jobs/pull/1039

Once this one is merged I believe we can safely close the issue

**Comment by ibuziuk** [comment]
have also create a separate issue for nighly builds of `arbitrary-users-patch` images https://github.com/eclipse/che/issues/13837

**Comment by nickboldt** [comment]
Assuming this is part of https://github.com/eclipse/che/issues/13557 this is a requirement for 7.0.0

**Comment by amisevsk** [comment]
Relevant PR: https://github.com/eclipse/che-devfile-registry/pull/38

**Comment by amisevsk** [comment]
Closing as I believe all components are complete. Please reopen if necessary.
